Doc/tutorial/appendix.rst

```diff
@@ -53,6 +53,32 @@ Typing an interrupt while a command is executing raises the
 :exc:`KeyboardInterrupt` exception, which may be handled by a :keyword:`try`
 statement.
 
+.. _tut-var:
+
+Special ``_`` variable
+-------------------------
+
+There is a special variable ``_`` in interactive mode, which stores the result of the last
+evaluated expression. If the last expression evaluated to ``None``, then the value of ``_``
+is not updated.
```

Comment:
```suggestion
There is a special variable ``_`` in interactive mode, which is updated with the result
of the last evaluated expression, unless it's ``None``.
```

Reply:
unless it was None?

(would like a native speaker to confirm!)